assets/js/main.js

```diff
@@ -57,4 +57,23 @@ $(document).ready(function() {
     loopwobble('#get-estimate',10,15000);
     loopwobble('#get-estimate-btn',10,20000);
 
-});
\ No newline at end of file
+});
+
+//Get every table class and append boostrap table-Responsive
+tables=document.getElementsByTagName("TABLE");//.className="table-responsive";
+
+for (var i = 0; i < tables.length; i++) {
+  //create a div wrapper
+  var wrapper = document.createElement('div');
+  // add table-responsive boostrap class
+  wrapper.className="table-responsive";
+  tables[i].parentNode.insertBefore(wrapper, tables[i]);
+  wrapper.appendChild(tables[i]);
+
+}
+
```

Comment:
loop through all images in content div of a blog and add `img-responsive center` class